presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConfig.java

```diff
@@ -45,6 +46,7 @@
     private boolean hideInternalColumns = true;
     private int messagesPerSplit = 100_000;
     private boolean timestampUpperBoundPushDownEnabled;
+    private SecurityProtocol securityProtocol = SecurityProtocol.PLAINTEXT;
```

Comment:
Please replicate the enum with supported values in Presto and use it here.

Reply:
will do